components/chainSelect/ChainSelect.tsx

```diff
@@ -135,7 +135,16 @@ const ChainSelect = () => {
       const firstAsset: ChainRegistryAsset | undefined = assetData.assets?.[0];
       const denom = firstAsset?.base || "";
       const displayDenom = firstAsset?.symbol || "";
-      const gasPrice = firstAsset ? `0.03${firstAsset.base}` : "";
+      const feeToken = chainData["fees"]?.fee_tokens.find(
+        (token: { denom: string }) => token.denom == denom,
+      );
```

Comment:
Here we also need a fallback denom I think (could be just `firstAsset.base` as before)

Reply:
The last "denom" variable in line 139 is defined in line 136 as `const denom = firstAsset?.base || "";`

Reply:
okay, but if `chainData["fees"]?.fee_tokens` is empty, the `find` will return `undefined`

Reply:
I just added an empty object fallback so the following lines don't fail, though